redis/redispipebp/breaker_test.go

```diff
@@ -7,9 +7,10 @@ import (
 	"time"
 
 	"github.com/joomcode/redispipe/redis"
-	"github.com/reddit/baseplate.go/breakerbp"
 	"github.com/sony/gobreaker"
 
+	"github.com/reddit/baseplate.go/breakerbp"
+
```

Comment:
We got _oooone_ leftover extra newline here

Reply:
hmmmmmmmmmmmmmmmm goland wachu doin